Spanner/src/Database.php

```diff
@@ -202,14 +204,15 @@ public function __construct(
         SessionPoolInterface $sessionPool = null,
         $returnInt64AsObject = false,
         array $info = [],
-        $databaseRole = null
+        $databaseRole = null,
+        $mapper = null
```

Comment:
let's name this variable `$valueMapper` for consistency

Reply:
I've renamed all the variables and options from `$mapper` -> `$valueMapper`.